Projects/Concrete_Strength_Regression/transform.py

```diff
@@ -84,6 +92,7 @@ def imputeMissing(*dfs: pd.DataFrame, columns: list = None, strategy: str) -> tu
                 df[column].fillna(df[column].agg(strategy), inplace=True)
     return dfs
 
+
 def standardScaleDataframe(X_train: pd.DataFrame, X_test: pd.DataFrame) -> tuple:
     """Fits and transforms train and test sets using StandardScaler
 
```

Comment:
One concept you might want to consider is using a Pipeline https://machinelearningmastery.com/modeling-pipeline-optimization-with-scikit-learn/